doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -97,21 +97,21 @@ Also note that Payara may utilize more than the default number of file descripto
 PostgreSQL
 ----------
 
```

Comment:
I'll just put his up here at the top. Please add a release note snippet.

Should it ask that developers switch to 16 or higher? 🤔

Reply:
I added this to the release note.